content/en/metrics/summary.md

```diff
@@ -42,18 +42,20 @@ The search bars provide the most comprehensive set of actions to filter the list
 {{< img src="metrics/summary/facets4.png" alt="Metrics Facet Panel" style="width:75%;">}}
 
 ## Configuration of multiple metrics 
-There are two buttons that allow you to configure multiple metrics at a time: 
 
-{{< img src="metrics/summary/configurationbuttons2.png" alt="Bulk Configuration Buttons" style="width:75%;">}}
+Clicking on **Configure Metrics** gives you three options that you can use to configure multiple metrics at a time: 
```

Comment:
```suggestion
Clicking on **Configure Metrics** gives you multiple options that you can use to configure more than one metric at a time: 
```

Reply:
I'm not necessarily opposed to this update, but rather curious as to what the rationale behind it.

Reply:
Basically, I try to avoid specific numbers whenever I can, because in future updates (say, adding something to the list of ways you can do it), changing the intro number might get missed.

Reply:
Thanks for asking! I sometimes forget that I should explain my reasoning for stuff like this. :)

Reply:
Thanks for clarifying!